src/omit.test.ts

```diff
@@ -26,5 +26,42 @@ test('type for curried form', () => {
 
   const result = omitFoo({ foo: 1, bar: 'potato' });
 
-  expectTypeOf(result).toEqualTypeOf<{ bar: string }>();
+  expectTypeOf(result).toEqualTypeOf<
+    Omit<Record<PropertyKey, unknown>, 'foo'>
+  >();
```

Comment:
Let's drop the whole test, it doesn't test anything that is meaningful for us to maintain.